### PR TITLE
ZEPPELIN-541 - increase logging in application

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
To increase logging in the application, so as it 

### What type of PR is it?
Refactoring

### Todos
* [x] - replace all printStackTrace() to LOGGER.error
* [x] - all catch block that doesn't throw exception, should have logging

### Is there a relevant Jira issue?
Yes, ZEPPELIN-541

### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Comment by jongyoul** [comment]
@prabhjyotsingh Thanks for the fixing this. It's good to me. But I think that changing the logger name from ```logger``` and ```LOG``` to ```LOGGER``` is irrelevant. Although Zeppelin doesn't get any policy to take a name for logger, it's better for now to reserve original name. Could you please fix this? Except that, LGTM.

**Comment by HeartSaVioR** [comment]
+1 to @jongyoul 
Before changing logger field name to be consistent, policy should be built first.

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/util/UtilsForTests.java [hunk 1]
```diff
@@ -110,10 +115,10 @@ public static void setEnv(String k, String v) {
 	            }
 	        }
 	      } catch (Exception e2) {
-	        e2.printStackTrace();
+					LOGGER.error(e2.toString());
```
Comment (corneadoug) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/util/UtilsForTests.java]: Looks like some indentation problems in both of these `catch`

Comment (prabhjyotsingh) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/util/UtilsForTests.java]: I have fixed this entire file, previously it had a lot of tab characters, now made it to all spaces ( tab size :2, Indent :2, Continuation indent :2)

**Comment by prabhjyotsingh** [comment]
@jongyoul, @HeartSaVioR thanks for the quick feed back, have reverted the variable name back to what they were.

**Comment by jongyoul** [comment]
@prabhjyotsingh Thanks for the quick fix. LGTM

**Comment by HeartSaVioR** [comment]
@prabhjyotsingh @jongyoul 
It would be better to also leave stack trace to logger message. 
`LOGGER.error(e.toString());` -> `LOGGER.error(e.toString(), e);`
What do you think?

**Comment by prabhjyotsingh** [comment]
@HeartSaVioR agreed, I was already half way doing this.

**Comment by jongyoul** [comment]
+1 @HeartSaVioR

**Comment by prabhjyotsingh** [comment]
Fixed CI, ready for review.

**Comment by jongyoul** [comment]
LGTM. Merging if there isn't no more discussion.

**Pull request closed without merging**